### PR TITLE
TL Localization - Chapter 7, stashing-cleaning.asc, submodules.asc (1,207 words)

### PR DESCRIPTION
**Pull request by heywens**
translation for:
* stashing-cleaning.asc - completed
* submodules.asc - partial (part 1)

### CODE REVIEW

File: book/07-git-tools/sections/stashing-cleaning.asc [hunk 1]
```diff
@@ -193,52 +193,52 @@ I-stash ang hunk na ito [y,n,q,a,d,/,e,?]? y
...
-If you want an easier way to test the stashed changes again, you can run `git stash branch <branch>`, which creates a new branch for you with your selected branch name, checks out the commit you were on when you stashed your work, reapplies your work there, and then drops the stash if it applies successfully:
+Kung ikaw ay nag-stash ng ilang trabaho, iwanan ito sa ilang saglit, at magpatuloy sa branch kung saan ikaw nag-stash ng trabaho, maaari kang magkaroon ng isang problema sa muling paglalapat ng trabaho.
+Kung sinusubukan ng paglalapat na baguhin ang isang file na binago mo na, makakakuha ka ng salungatan sa pagsasama-sama at kakailanganin mong lutasin ito.
+Kung gusto ang isang mas madaling paraan upang suriing muli ang mga naka-stash na pagbabago, maaari mong patakbuhin ang `git stash branch <branch>`, kung saan ay lumilikha ng isang bagong branch para sa iyo kasama ang iyong napiling pangalan ng branch, nagsusuri sa iyong commit kapag ikaw ay nag-stash ng iyong trabaho, muling naglalapat sa iyong trabaho doon, at pagkatapos ay naghuhulog sa stash kung ito ay matagumpay na nailalapat:
```
Comment (mind-blast) [inline book/07-git-tools/sections/stashing-cleaning.asc]: para sa iyo kasama ang iyong napiling pangalan ng branch to para sa iyong napiling pangalan ng branch

File: book/07-git-tools/sections/stashing-cleaning.asc [hunk 2]
```diff
@@ -247,9 +247,9 @@ Would remove test.o
 Would remove tmp/
 ----
 
-By default, the `git clean` command will only remove untracked files that are not ignored.
-Any file that matches a pattern in your `.gitignore` or other ignore files will not be removed.
-If you want to remove those files too, such as to remove all `.o` files generated from a build so you can do a fully clean build, you can add a `-x` to the clean command.
+Bilang default, ang utos na `git clean` ay tinatanggal lamang ang mga hindi sinubaybayan na mga file na hindi binabalewala.
+Anumang file na tumutugma sa isang pattern sa iyong `.gitignore` o ibang hindi pansinin ng mga file ay hindi matatanggal.
```
Comment (mind-blast) [inline book/07-git-tools/sections/stashing-cleaning.asc:251]: pattern to hulmahan

File: book/07-git-tools/sections/stashing-cleaning.asc [hunk 3]
```diff
@@ -247,9 +247,9 @@ Would remove test.o
...
+Kung gusto mo na tanggalin din ang mga file na ito, tulad ng tanggalin lahat ang mga file na `.o` na binuo mula sa isang build upang makakagawa ka ng buong malinis na build, maaari kang magdagdag ng `-x` sa clean na utos.
```
Comment (mind-blast) [inline book/07-git-tools/sections/stashing-cleaning.asc:252]: clean to malinis

File: book/07-git-tools/sections/stashing-cleaning.asc [hunk 4]
```diff
@@ -259,36 +259,36 @@ $ git status -s
...
-Would remove test.o
-Would remove tmp/
+Magtanggal build.TMP
+Magtanggal test.o
+Magtanggal tmp/
 ----
 
-If you don't know what the `git clean` command is going to do, always run it with a `-n` first to double check before changing the `-n` to a `-f` and doing it for real.
-The other way you can be careful about the process is to run it with the `-i` or ``interactive'' flag.
+Kung hindi mo alam kung ano ang gagawin ng command na `git clean`, palaging patakbuhin ito ng may isang` -n` muna upang mag-double check bago baguhin ang `-n` sa isang` -f` at gagawain ito sa totoo.
```
Comment (mind-blast) [inline book/07-git-tools/sections/stashing-cleaning.asc]: command to utos

Comment (mind-blast) [inline book/07-git-tools/sections/stashing-cleaning.asc]: mag-double check to dobleng pagsuri

File: book/07-git-tools/sections/stashing-cleaning.asc [hunk 5]
```diff
@@ -259,36 +259,36 @@ $ git status -s
...
+Ang iba pang mga paraan na maaari kang maging maingat tungkol sa proseso ay ang patakbuhin ito sa flag ng `-i` o ``interactive''.
 
-This will run the clean command in an interactive mode.
+Papatakbuhin nito ang clean na uto sa isang interactive na mode.
```
Comment (mind-blast) [inline book/07-git-tools/sections/stashing-cleaning.asc]: clean to malinis
uto to utos

File: book/07-git-tools/sections/stashing-cleaning.asc [hunk 6]
```diff
@@ -259,36 +259,36 @@ $ git status -s
...
 $ git clean -x -i
-Would remove the following items:
+Magtanggal ng mga sumusunod na mga aytem:
   build.TMP  test.o
-*** Commands ***
-    1: clean                2: filter by pattern    3: select by numbers    4: ask each             5: quit
-    6: help
-What now>
+*** Mga kautusan ***
+    1: clean                2: mag-filter gamit ang pattern    3: pumili ng mga numero    4: tanungin ang bawat isa             5: huminto
```
Comment (mind-blast) [inline book/07-git-tools/sections/stashing-cleaning.asc:282]: pattern to hulmahan

File: book/07-git-tools/sections/submodules.asc [hunk 7]
```diff
@@ -1,61 +1,61 @@
 [[_git_submodules]]
-=== Submodules
+=== Mga Submodule
 
-It often happens that while working on one project, you need to use another project from within it.
-Perhaps it’s a library that a third party developed or that you’re developing separately and using in multiple parent projects.
-A common issue arises in these scenarios: you want to be able to treat the two projects as separate yet still be able to use one from within the other.
+Madalas na nangyayari na habang nagtatrabaho sa isang proyekto, kailangan mong gumamit ng isa pang proyekto mula sa loob nito.
+Marahil ito ay isang library na binuo ng isang third party o ikaw ay hiwalay lumilikha at ginagamit sa maraming mga proyektong magulang.
```
Comment (mind-blast) [inline book/07-git-tools/sections/submodules.asc:5]: third party to ikatlong partido

File: book/07-git-tools/sections/submodules.asc [hunk 8]
```diff
@@ -1,61 +1,61 @@
...
-	new file:   .gitmodules
-	new file:   DbConnector
+	bagong file:   .gitmodules
+	bagong file:   DbConnector
 ----
 
-First you should notice the new `.gitmodules` file.
-This is a configuration file that stores the mapping between the project’s URL and the local subdirectory you’ve pulled it into:
+Una dapat mong mapansin ang bagong file na `.gitmodules`.
+Ito ay isang file ng pagsasaayos na nag-iimbak sa pag-map sa pagitan ng URL ng proyekto at ang lokal na subdirectory na na-pull mo ito:
```
Comment (mind-blast) [inline book/07-git-tools/sections/submodules.asc]: pag-map to pagmapa

File: book/07-git-tools/sections/submodules.asc [hunk 9]
```diff
@@ -64,17 +64,17 @@ This is a configuration file that stores the mapping between the project’s URL
...
 
-[NOTE]
+[TANDAAN]
 =====
-Since the URL in the .gitmodules file is what other people will first try to clone/fetch from, make sure to use a URL that they can access if possible.
-For example, if you use a different URL to push to than others would to pull from, use the one that others have access to.
-You can overwrite this value locally with `git config submodule.DbConnector.url PRIVATE_URL` for your own use.
-When applicable, a relative URL can be helpful.
+Dahil ang URL sa file na .gitmodules ay kung ano ang unang subukan ng mga tao upang i-clone/i-fetch, siguraduhin na gumamit ng isang URL na maaari nilang ma-access kung posible.
+Halimbawa, kung gumamit ka ng isang iba't ibang URL upang mag-push kaysa sa iba na gusto na mag-pull, gumamit ng isa na may access ang iba.
```
Comment (mind-blast) [inline book/07-git-tools/sections/submodules.asc:75]: access to pahintulot

File: book/07-git-tools/sections/submodules.asc [hunk 10]
```diff
@@ -64,17 +64,17 @@ This is a configuration file that stores the mapping between the project’s URL
...
+Maaari mong i-overwrite ang halaga na ito nang lokal gamit ang `git config submodule.DbConnector.url PRIVATE_URL` para sa pansariling paggamit.
```
Comment (mind-blast) [inline book/07-git-tools/sections/submodules.asc]: i-overwrite to sapawan

**Comment by mind-blast** [comment]
Please check on my suggested words. Thank you.

**Comment by heywens** [comment]
Hi Sir @robin-ho, thanks for the review.
I have applied above suggestions except translating the following words:
* pattern
* third party

Thanks!

**Pull request merged**